libsolidity/grammar.txt

```diff
@@ -79,7 +79,7 @@ Expression =
 
 PrimaryExpression = Identifier | BooleanLiteral | NumberLiteral | StringLiteral
 
-FunctionCall = Identifier '(' Expression? ( ',' Expression )* ')'
+FunctionCall = ( Identifier | MemberAccess ) '(' Expression? ( ',' Expression )* ')'
```

Comment:
From the parser code, function calls are a bit more complicated:

It should rather be something like:

PrimaryExpression | NewExpression | TypeName

and on top of this, you can iteratively do MemberAccess (restricted to such base expressions), IndexAccess (again restricted) and the actual function call.